docs/docs/mlflow-3/index.mdx

```diff
@@ -0,0 +1,341 @@
+        ],
+    }
+)
+
+# Start a run to represent the evaluation job
+with mlflow.start_run() as evaluation_run:
+    eval_dataset = mlflow.data.from_pandas(
+        df=eval_df, name="eval_dataset", targets="expected_response"
+    )
+    # Run the evaluation based on extra metrics
+    result = mlflow.evaluate(
+        model=f"models:/{logged_model.model_id}",
+        data=eval_dataset,
+        # targets="expected_response",
+        model_type="question-answering",
+        extra_metrics=[
+            mlflow.metrics.latency(),
+            mlflow.metrics.genai.answer_correctness("openai:/gpt-4o-mini"),
+        ],
+        # This is needed since answer_correctness looks for 'inputs' field
+        evaluator_config={"col_mapping": {"inputs": "messages"}},
+    )
+
+print(result.tables["eval_results_table"])
+#                                         messages                                  expected_response  ... answer_correctness/v1/score                answer_correctness/v1/justification
+# 0  What is MLflow Tracking and how does it work?  MLflow Tracking is a key component of the MLfl...  ...                           5  The output accurately describes MLflow Trackin...
+# 1                         What is Unity Catalog?  Unity Catalog is a feature in Databricks that ...  ...                           5  The output accurately describes Unity Catalog ...
+# 2        What are user-defined functions (UDFs)?  User-defined functions (UDFs) in the context o...  ...                           5  The output accurately describes user-defined f...
+```
+
+
+## Migration Guide
+
+MLflow 3.0 introduces some key API changes while also removes some outdated features. This guide will help you transition smoothly to the latest version.
+
+### Key changes
+
+- `mlflow.<flavor>.log_model` API usage: `artifact_path` parameter is deprecated, use **`name`** instead
```

Comment:
Can we show a before / after comparison with code here to make it more clear to users what is changing?